File: Template/FindDuplicates.java

boolean findDuplicates(List<Type>& type) {

    Set<Type hanhset = new HashSet<>();
    for (Type key: keys) {
        if (hashset.contains(key)) { 
            return true;
        }
        hashset.insert(key);
    }
    return false;
}

File: Template/FindDuplicates.cpp
bool findDuplicate(vector<Type>& keys) {
    unordered_set<Type> hashset;
    for(Type key : keys) { 
        if (hashset.count(key) > 0 ) {
            return true;
        }
        hashset.insert(key);
    }
    return false;
}
